docs/cli.md

```diff
@@ -1035,9 +1040,9 @@ viam.service.vision.v1.VisionService.GetClassificationsFromCamera
 | -------- | ----------- | ------------------- | --------- |
 | `--organization` | Organization name that the machine belongs to | `list`, `status`, `logs`, `part` | **Required** |
 | `--location` | Location name that the machine belongs to | `list`, `status`, `logs`, `part` | **Required** |
-| `--robot` | Machine ID for which the command is being issued | `status`, `logs`, `part` | **Required** |
+| `--machine` | Machine ID for which the command is being issued | `status`, `logs`, `part`, `part restart` | **Required** |
 | `--errors` | Boolean, return only errors (default: false) | `logs` | Optional |
-| `--part` | Part name for which the command is being issued | `logs` | Optional |
+| `--part` | Part name (`logs`) or ID (``part restart`) for which the command is being issued | `logs`, `part restart` | Optional |
```

Comment:
`--machine` and `--part` is required for all `part` subcommands:
```
➜  ~ viam robot part
NAME:
   viam machines part - work with a machine part

USAGE:
   viam machines part command [command options] [arguments...]

COMMANDS:
   status     display part status
   logs, log  display part logs
   restart    request part restart
   run        run a command on a machine part
   shell      start a shell on a machine part
   cp         copy files to and from a machine part
```

So it feels a bit strange to list just one of them here? (Not sure if it makes sense in context)

Reply:
that makes sense, I'll change to just "part"!